src/algebra/ring.lean

```diff
@@ -158,6 +158,19 @@ by simp
   (if P then 1 else 0) * a = if P then a else 0 :=
 by simp
 
+@[to_additive]
+lemma ite_mul_one {α : Type*} [monoid α] {P : Prop} [decidable P] {a b : α} :
```

Comment:
Please move this one to `algebra/group`.

Reply:
Done!